docker-compose.yaml

```diff
@@ -1,27 +1,27 @@
-version: '3'
+version: '3.5'
 
 services:
+  lab:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        PYTHON_VERSION: ${PYTHON_VERSION:-3.9}
+    volumes:
+      - .:/home/rigetti/pyquil/
+    environment:
+      - QCS_SETTINGS_APPLICATIONS_PYQUIL_QVM_URL=http://qvm:5000
+      - QCS_SETTINGS_APPLICATIONS_PYQUIL_QUILC_URL=tcp://quilc:5555
```

Comment:
These environment variables should be `QCS_APPLICATIONS_QVM_URL` `QCS_APPLICATIONS_QUILC_URL`

Reply:
Good point. Will update since targeting v4